modules/src/mock/context.rs

```diff
@@ -59,11 +67,30 @@ pub struct MockContext {
     /// All the connections in the store.
     connections: HashMap<ConnectionId, ConnectionEnd>,
 
+    /// All the channels in the store. TODO Make new key PortId X ChanneId
+    channels: HashMap<(PortId, ChannelId), ChannelEnd>,
```

Comment:
Since we use the `(PortId, ChannelId)` tuple so often, it may be a good idea to address this comment and add a struct that captures the tuple.

Something like either of the two:
```rust
// A tuple struct
struct OwnedChannelId(PortId, ChannelId);

// A struct with two fields
struct OwnedChannelId {
    port_id: PortId,
    channel_id: ChannelId,
}
```

Reply:
yes :) we had a chat on this and you suggest the pair since is faster: and it was :).

Reply:
Thanks for reminding me, that makes sense. We'll fix this later on then!